### PR TITLE
Newsletter 11: Add Legion Jam

### PR DESCRIPTION
**Pull request by ozkriff**
Part of #172 (_"Newsletter 11: Coordination/Tracking"_) 

@laticoda r?

------

image

### CODE REVIEW

**Comment by laticoda** [comment]
thank you for including it in newsletter !

File: content/posts/newsletter-011/index.md [hunk 1]
```diff
@@ -60,6 +61,35 @@ some actual interesting discussions.
 If needed, a section can be split into subsections with a "------" delimiter.
 -->
 
+## [Legion Game Jam][legion-jam]
+
+[![The Roman Empire at its greatest extent](roman-empire.jpeg)][roman-empire-wiki]
+
+The "Legion" game jam by [Laticoda] is about
+[Roman Empire][roman-empire-wiki] for the Background Thema
```
Comment (17cupsofcoffee) [inline content/posts/newsletter-011/index.md:69]: Guessing this is meant to be:

```suggestion
the [Roman Empire][roman-empire-wiki] for the background theme
```

File: content/posts/newsletter-011/index.md [hunk 2]
```diff
@@ -60,6 +61,35 @@ some actual interesting discussions.
...
+and ECS paradigm for the technical side.
+The conditions are:
+
+- Open-source & Rust only.
+- Projects should use some ECS library
+  (you can choose [Legion][legion] if you don't have one).
+- Team working and recycling old spinets are allowed.
```
Comment (17cupsofcoffee) [inline content/posts/newsletter-011/index.md:76]: 'spinets'?

File: content/posts/newsletter-011/index.md [hunk 3]
```diff
@@ -60,6 +61,35 @@ some actual interesting discussions.
...
+
+> Submissions open to August 1st 2020.
+>
+> It can be RPG, strategic, arcade or else; multi or solo. What you want.
+> Just try to include a little bit history and culture taste.
+> Dont be afraid, it is not ranked.
+
+Also, participants are encouraged to
+[document the developemnt process at the event's forum][legion-jam-forum].
```
Comment (17cupsofcoffee) [inline content/posts/newsletter-011/index.md:85]:
```suggestion
[document the development process at the event's forum][legion-jam-forum].
```

**Pull request merged**